src/arrayMethodJoin.js

```diff
@@ -3,9 +3,22 @@
 /**
  * Implement method join
  */
+
 function applyCustomJoin() {
-  [].__proto__.join2 = function(separator) {
-    // write code here
+  [].__proto__.join2 = function(separator = ',') {
+    let result = '';
```

Comment:
result for what? This name doesn’t describe your variable

Reply:
function result

Reply:
and the result is a string

Reply:
> and the result is a string
When I see result name, I don’t understand what is it

Reply:
I changed to stringOutcome. Is it good?

Reply:
😂
no